test/components/CalendarDay_spec.jsx

```diff
@@ -44,6 +44,10 @@ describe('CalendarDay', () => {
       handleDayClickSpy = sinon.spy(CalendarDay.prototype, 'handleDayClick');
     });
 
+    afterEach(() => {
+      CalendarDay.prototype.handleDayClick.restore();
```

Comment:
we should probably use sinon-sandbox instead of having to restore spies after every test manually